backend/app/routers/volunteers.py

```
@@ -153,24 +222,32 @@ async def update_volunteer(
 
     await db.commit()
     await db.refresh(volunteer)
-    return _to_volunteer_out(volunteer)
+    periods_map = await _load_periods_map(db, [volunteer.id])
+    return _to_volunteer_out(volunteer, periods_map.get(volunteer.id, []))
 
 
 @router.delete("/{volunteer_id}", status_code=status.HTTP_204_NO_CONTENT)
-async def delete_volunteer(volunteer_id: str, db: AsyncSession = Depends(get_db)) -> None:
+async def delete_volunteer(
+    volunteer_id: str, db: AsyncSession = Depends(get_db)
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)